README.rst

```diff
@@ -2,108 +2,260 @@
+
+When editing a pattern (e.g. in ``mockup`` package), it's important to know for which bundle it's registered, which can
+be done by checking where it's imported in the entry points above. The bundle name will be useful as parameter for
+``plone-compile-resources`` script.
+
+.. note::
+  Patterns are also available as independent resources in ``Resource Registries`` control panel but they are not
+  registered to default Plone bundles, which instead use the entry point approach. Pattern's resources may be useful if
+  needed to be injected in specific views.
+
+In production mode (``./bin/instance start``), since version 5.1, Plone is configured to serve Aggregate Bundles,
+single files to minimize network requests (`aggregate bundles
+<https://docs.plone.org/adapt-and-extend/theming/resourceregistry.html#resource-bundle-aggregation>`_).
+In that case final production resource URLs will be similar to:
+
+- http://localhost:8080//++plone++production/++unique++TIMESTAMP/default.js
+- http://localhost:8080//++plone++production/++unique++TIMESTAMP/default.css
+
+If you enable Development Mode for JavaScript and CSS in Resource Registries control panel, Plone will omit timestamp
+from path and serve fresh copies of the resources of the selected bundle(s). Those are compiled in-browser, on the fly
+for each page load and requested by XHR requests to URLs like:
 
-- Increase ``npm`` package versions in ``package.json``, in sections ``dependencies`` or ``devDependencies``.
+- http://localhost:8080/++resource++plone.js
+- http://localhost:8080/++plone++static/plone.less
+- http://localhost:8080/++resource++plone-logged-in.js
+- http://localhost:8080/++plone++static/plone-logged-in.less
+
+These entry points will then cause Patterns themselves to be loaded through in-browser XHR requests to URLs like:
+
+- http://localhost:8080/Plone/++resource++mockup/livesearch/pattern.js
+- http://localhost:8080/Plone/++resource++mockup/livesearch/pattern.livesearch.less
+
+If you do enable Development Mode, and yet do not select any bundles, Plone serves static resources in URLs similar to:
+
+- http://localhost:8080/++plone++static/++unique++TIMESTAMP/plone-compiled.min.js
+- http://localhost:8080/++plone++static/++unique++TIMESTAMP/plone-compiled.css
+- http://localhost:8080/++plone++static/++unique++TIMESTAMP/plone-logged-in-compiled.min.js
+- http://localhost:8080/++plone++static/++unique++TIMESTAMP/plone-logged-in-compiled.css
+
+
+Development tips
+^^^^^^^^^^^^^^^^
+
+When fixing Plone bugs or improving functionality:
+
+- Dependencies upgrades (e.g. ``moment.js``) should be done in ``mockup`` and after the PR is merged, propagated here
+  (i.e. static resources needs to be recompiled)
+- Likewise, CSS and JS should be developed not in ``plone.staticresources`` but in their own package's source (for
+  instance, in ``mockup``)
+- To see the new changes in the browser, enable ``Development Mode`` in ``Resource Registries`` control panel. Next the
```

Comment:
Perhaps use the [`:guilabel:` directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-guilabel) for "Development Mode" and "Resource Registries"?